install_config/topics/sssd_for_ldap_failover_configure_sssd.adoc

```diff
@@ -0,0 +1,92 @@
+////
+configuring sssd for ldap failover and extended attributes
+
+Module included in the following assemblies:
+
+* install_config/sssd_for_ldap_failover.adoc
+////
+
+[id='sssd-configuring-sssd-{context}']
+= Configuring SSSD for LDAP failover
+Complete these steps on the remote basic authentication server.
+
+You can configure the SSSD to retrieve attributes, such as email addresses and 
+display names, and pass them to {product-title} to display in the web interface.
+In the following steps, you configure the SSSD to provide email addresses to
+{product-title}:
+
+. Install the required SSSD and the web server components:
++
+----
+# yum install -y sssd \
+                 sssd-dbus \
+                 realmd \
+                 httpd \
+                 mod_session \
+                 mod_ssl \
+                 mod_lookup_identity \
+                 mod_authnz_pam \
+                 php \
+                 mod_php
+----
+
+. Edit the *_/etc/httpd/conf.modules.d/55-authnz_pam.conf_* file and uncomment
+ the following line:
++
+----
+LoadModule authnz_pam_module modules/mod_authnz_pam.so
+----
+
+. Set up SSSD to authenticate this VM against the LDAP server. If the LDAP server
+is a FreeIPA or Active Directory environment, then use *realmd* to join
+this machine to the domain.
++
+----
+# realm join ldap.example.com
+----
++
+For more advanced cases, see the
+https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/System-Level_Authentication_Guide/authconfig-ldap.html[System-Level Authentication Guide]
+
+. To use SSSD to manage failover situations for LDAP, add more entries to the
+ *_/etc/sssd/sssd.conf_* parameter on the *ldap_uri* line. Systems that are 
+enrolled with FreeIPA can automatically handle failover by using DNS SRV records.
+
+. Modify the *[domain/DOMAINNAME]* section of the *_/etc/sssd/sssd.conf_* file
+and add this attribute:
++
+----
+[domain/example.com]
+...
+ldap_user_extra_attrs = mail
+----
+
+. Grant Apache permission to retrieve the email attribute. Add the following 
+lines of code to the *[ifp]* section of the *_/etc/sssd/sssd.conf_* file:
++
+----
+[ifp]
+user_attributes = +mail
+allowed_uids = apache, root
+----
+
+. To ensure that all of the changes are applied properly, restart SSSD:
++
+----
+$ systemctl restart sssd.service
+----
+
+. Test that the user information can be retrieved properly:
++
+----
+$ getent passwd <username>
+username:*:12345:12345:Example User:/home/username:/usr/bin/bash
+----
+
+. Attempt to log into the VM as an LDAP user and confirm that you can log in
+using LDAP credentials. You can use either the local console or a remote service
+like SSH to log in.
+
+. If you do not want users to log in to the remote basic authentication server by
+using LDAP, edit the *_/etc/pam.d/system-auth_* and
+*_/etc/pam.d/password-auth_*  files and remove the lines that contain *pam_sss.so*.
```

Comment:
Should probably advice to create proper HBAC rules for IPA, or GPOs for AD.
Removing lines from those files will be reversed during upgrades. It is ok to point to SSSD documentation for futher information instead of providing half-broken guidance here.